src/Tizen.NUI/src/public/BaseComponents/TextEvent.cs

```diff
@@ -23,15 +23,13 @@ namespace Tizen.NUI.BaseComponents
     /// <summary>
     /// AnchorClickedEventArgs is a class to record anchor click event arguments which will sent to user.
     /// </summary>
-    /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 9 </since_tizen>
     public class AnchorClickedEventArgs : EventArgs
     {
         /// <summary>
         /// Anchor href.
```

Comment:
It should be enough as almost everybody know what href is, but consider adding expanded abbreviation in parenthesis.

Reply:
Done